.github/workflows/full_test.yml

```diff
@@ -26,7 +27,7 @@ jobs:
     - name: Set up .NET
       uses: actions/setup-dotnet@v3
       with:
-        dotnet-version: '5.0'
+        dotnet-version: '3.1.x'
```

Comment:
.NET 3.1 reached end-of-life in 2022😳

Reply:
@ecomodeller I updated the runtimeconfig.json to use latest available .NET runtime and setup-dotnet@v3 to use .NET 6.0. Apparently was quite simple :-).